extras/kms/repository_root_key_version.go

```diff
@@ -174,3 +176,96 @@ func (r *repository) ListRootKeyVersions(ctx context.Context, keyWrapper wrappin
 	}
 	return versions, nil
 }
+
+// rewrapRootKeyVersionsTx will rewrap (re-encrypt) the root key versions for a
+// given rootKeyId with the latest wrapper.
+// This function encapsulates all the work required within a dbw.TxHandler and
+// allows this capability to be shared with other repositories or just called
+// within a transaction.  To be clear, this repository function doesn't include
+// its own transaction and is intended to be used within a transaction provide
```

Comment:
```suggestion
// its own transaction and is intended to be used within a transaction provided
```

Reply:
(And elsewhere this was copied and pasted)

Reply:
ty